facttic/recumap-frontend
# Feature/auth provider

This PR provides:

+ A simple auth provider with a login service
+ A simple data provider using axios

How to test:

+ Get the backend server running
+ Run the development frontend in localhost:3000
+ Create a user in the backend if you didn't
+ Login with credentials

Expected result: you should log in succesfully and be redirected to the home page

Closes #4 
Closes #5
